### PR TITLE
Bug 2051320: Update OLM metadata to 4.11

### PR DESCRIPTION
**Pull request by jsafrane**
cc @openshift/storage

### CODE REVIEW

File: config/manifests/4.11/aws-efs-csi-driver-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ metadata:
     repository: https://github.com/openshift/aws-efs-csi-driver-operator
     createdAt: "2021-07-14T00:00:00Z"
     description: Install and configure AWS EFS CSI driver.
-    olm.skipRange: ">=4.9.0 <4.10.0"
+    olm.skipRange: ">=4.11.0 <4.11.0"
```
Comment (bertinatto) [inline config/manifests/4.11/aws-efs-csi-driver-operator.clusterserviceversion.yaml]: Isn't this self-contradicting?

Comment (jsafrane) [reply config/manifests/4.11/aws-efs-csi-driver-operator.clusterserviceversion.yaml]: It is :-).

Looking at LSO, the min. version should be the lowest where the operator was published, fixed.

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**